FoodApp/Models/Dish.cs

```diff
@@ -8,9 +9,13 @@ namespace FoodApp.Models
     public class Dish
     {
         public int Id { get; set; }
+        [Required]
         public string Name { get; set; }
+        [Required]
         public string Details { get; set; }
         public double Price { get; set; }
```

Comment:
is Price not required for a dish? Also there should some `[Range()]` param set too. Negative prices are not very common :^)

Reply:
Will look into it more in next commits :)